client/js/views/Home.vue

```diff
@@ -91,9 +92,9 @@ export default {
     };
   },
   computed: {
-    gameList() {
-      return this.$store.state.gameList.games;
-    },
+    ...mapState({
```

Comment:
One small change, switch this to use destructuring like we do elsewhere in the codebase— otherwise this looks great!
```
...mapState({
  waitingForOpponentToPlayFromDeck: ({ game }) => game.waitingForOpponentToPlayFromDeck,
}),
```

Reply:
Thanks for the comment, makes sense.
It's fixed within 0958361874b64c7338112cc50b8f77782affc6c8
Please, let me know if I need to squash the commits.

Reply:
And let me know if I need to fix the issuer linter has mentioned [here](https://github.com/cuttle-cards/cuttle/actions/runs/3210377671/jobs/5247779851)
That place seem to be related to my previous PR https://github.com/cuttle-cards/cuttle/pull/137 but I did there nothing but replaced the string.

Reply:
Perfect thank you. I should have run CI before merging the string change. Looks like it crossed the line length limit with that small tweak; thanks for addressing it